packages/renderer/server/render-video.ts

```diff
@@ -52,6 +53,8 @@ export interface RenderVideoSettings {
   workers?: number;
   dimensions?: [number, number];
   logProgress?: boolean;
+  outDir?: string;
```

Comment:
This object is getting quite messy. Let's restructure it and do a breaking release. I'm also not sure whether I like outDir and name being two different params. Maybe we can put them into an object like `{ out: { file, workingDirectory} }` or something like that.